core/controllers/profile_test.py

```diff
@@ -312,3 +312,60 @@ def test_contribution_datetime(self):
         self.assertEqual(
             response_dict['first_contribution_datetime'],
             utils.get_time_in_millisecs(current_datetime))
+
+
+class UserContributionsTests(test_utils.GenericTestBase):
+
+    USERNAME_A = 'a'
+    EMAIL_A = 'a@example.com'
+    USERNAME_B = 'b'
+    EMAIL_B = 'b@example.com'
+    EXP_ID_1 = 'exp_id_1'
+
+    def test_zero_count(self):
+        #Check to see a user with no contributions
+        #shows up as 0 created and edited on the profile page.
+        self.signup(self.EMAIL_A, self.USERNAME_A)
+        self.login(self.EMAIL_A)
+        response_dict = self.get_json(
+            '/profilehandler/data/%s' % self.USERNAME_A)
+        self.assertEqual(response_dict['created_explorations_count'], 0)
+        self.assertEqual(response_dict['edited_explorations_count'], 0)
+
+    def test_created_count(self):
+        #Check to see a user with 1 created explorations
+        #shows up as 1 created and edited on the profile page.
+        self.signup(self.EMAIL_A, self.USERNAME_A)
+        self.login(self.EMAIL_A)
+        self.user_a_id = self.get_user_id_from_email(self.EMAIL_A)
+        self.save_new_valid_exploration(
+                self.EXP_ID_1, self.user_a_id, end_state_name='End')
+        response_dict = self.get_json(
+            '/profilehandler/data/%s' % self.USERNAME_A)
+        self.assertEqual(response_dict['created_explorations_count'], 1)
+        self.assertEqual(response_dict['edited_explorations_count'], 1)
+
+    def test_edited_count(self):
+        #Check to see a user with 1 edited exploration
+        #shows up as 1 edited on the profile page.
+        self.signup(self.EMAIL_A, self.USERNAME_A)
+        self.user_a_id = self.get_user_id_from_email(self.EMAIL_A)
+
+        self.signup(self.EMAIL_B, self.USERNAME_B)
+        self.login(self.EMAIL_B)
+        self.user_b_id = self.get_user_id_from_email(self.EMAIL_B)
+
+        self.save_new_valid_exploration(
+            self.EXP_ID_1, self.user_a_id, end_state_name='End')
+
+        exp_services.update_exploration(self.user_b_id, self.EXP_ID_1, [{
+            'cmd': 'edit_exploration_property',
+            'property_name': 'objective',
+            'new_value': 'the objective'
+            }], 'Test edit')
```

Comment:
So if the line is too long, should it be in by 4? Should the indentation on this one be aligned with exp_services?